### PR TITLE
OCPBUGS-35898 Adding CAPA DNS timeout known issue

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.16

Issue:
https://issues.redhat.com/browse/OCPBUGS-35898

Link to docs preview:

QE review:
- [ ] QE has approved this change.
Due to time zones, QE will not be able to review until post merge.

### CODE REVIEW

**Review by r4f4 — COMMENTED** [review]
/lgtm

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -3148,6 +3148,8 @@ In the following tables, features are marked with the following statuses:
 
 * Installing {product-title} {product-version} with the Infrastructure Operator, Central Infrastructure Management, or ZTP methods using ACM versions 2.10.3 or earlier is not possible. This is because of a change in the dynamically linked installer binary,`openshift-baremetal-install`, which in {product-title} {product-version} requires a Red Hat Enterprise Linux (RHEL) 9 host to run successfully. It is planned to use the statically linked binary in future versions of ACM to avoid this issue. (link:https://issues.redhat.com/browse/ACM-12405[*ACM-12405*])
 
+* When installing a cluster on {aws-short}, the installation can timeout if the load balancer DNS time-to-live (TTL) value is very high. (link:https://issues.redhat.com/browse/OCPBUGS-35898[*OCPBUGS-35898*])
```
Comment (skopacz1) [inline release_notes/ocp-4-16-release-notes.adoc:3151]: Should this be two words?
```suggestion
* When installing a cluster on {aws-short}, the installation can time out if the load balancer DNS time-to-live (TTL) value is very high. (link:https://issues.redhat.com/browse/OCPBUGS-35898[*OCPBUGS-35898*])
```

**Review by skopacz1 — COMMENTED** [review]
Just one question otherwise LGTM!

File: release_notes/ocp-4-16-release-notes.adoc (hunk 1 above)
Comment (bscott-rh) [reply release_notes/ocp-4-16-release-notes.adoc:3151]: Yup, I think "time out" is better here.

**Pull request merged**